.eslintrc.js

```diff
@@ -22,6 +22,13 @@ module.exports = {
       parserOptions: {
         project: './tsconfig.json',
       },
-    }
+    },
+    {
+      files: ['*.tsx'],
+      rules: {
+        // Turn off jsdoc on tsx files until jsdoc is fixed for tsx files
```

Comment:
Is there an issue tracking this bug you could link here? That way someone coming upon this months or years from now will easily be able to check if the restriction is still valid.